index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kalam">
+  <link rel="stylesheet" href="css/style.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <div class="header">
+      <ul>
+        <li id="time">
+          <a href="#">12:14:45</a>
+        </li>
+        <li>
+          <a href="#">BEST PIZZA</a>
+        </li>
+        <li class="active">
+          <a href="#">SIGN UP</a>
+        </li>
+      </ul>
+    </div>
+  </header>
+
+  <main>
+
+    <div class="row">
```

Comment:
`row` means using of `col`s inside. But here it looks more like `container`. Am I right?

If so, make some changes.